src/feature/BoltFeature.ts

```diff
@@ -0,0 +1,35 @@
+import { ExtensionContext, commands, window, Uri } from 'vscode';
+import { IFeature } from '../feature';
+import * as path from 'path';
+import * as fs from 'fs';
+
+export class BoltConfigFeature implements IFeature {
+  dispose() {}
+  constructor(context: ExtensionContext) {
+    context.subscriptions.push(
+      commands.registerCommand('puppet-bolt.OpenUserConfigFile', () => {
+        let userInventoryFile = path.join(process.env['USERPROFILE'] || '~', '.puppetlabs', 'bolt', 'bolt.yml');
+
+        if (!fs.existsSync(userInventoryFile)) {
+          window
+            .showQuickPick(['yes', 'no'], {
+              placeHolder: `Default bolt config yml not present. Do you want to create it?`,
+              canPickMany: false,
+              ignoreFocusOut: true
+            })
+            .then(answer => {
+              switch (answer) {
+                case 'no':
+                  break;
+                case 'yes':
+                  fs.writeFile(userInventoryFile, '', function(err) {});
```

Comment:
Do we need to specify UTF8 encoding here? or add boilerplate examples commented out?

Reply:
UTF8 good idea.

Boilerplate good idea, but better in the snippets than here

Reply:
So I added utf8, then a short commented out line pointing people to use the snippets